swiftpkg/internal/bzl_selects.bzl

```diff
@@ -155,34 +160,33 @@ def _to_starlark(values, kind_handlers = {}, mutually_inclusive = False):
     # dict whose keys are the conditions and the value is the value for the
     # condition.
     selects_by_kind = {}
-    no_condition_results = sets.make()
+    no_condition_results = []
```

Comment:
Why did you move away from the set? Are we sure that duplicates are not possible?

Reply:
I just re-read the description on the PR. I will approve.